README.rst

```diff
@@ -72,3 +67,219 @@ response, e.g.
+  bucket.
+* Allow the internet traffic to access Wagtail image renditions (``images/*``).
+
+The user permissions can be set in the IAM or via a bucket policy. See example
+of all of those points being achieved with a bucket policy below.
+
+.. code:: json
+
+   {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Sid": "PublicGetObject",
+                "Effect": "Allow",
+                "Principal": "*",
+                "Action": "s3:GetObject",
+                "Resource": "arn:aws:s3:::[BUCKET NAME]/images/*"
+            },
+            {
+                "Sid": "AllowUserManageBucket",
+                "Effect": "Allow",
+                "Principal": {
+                    "AWS": "arn:aws:iam::[USER ARN]"
+                },
+                "Action": [
+                    "s3:ListBucket",
+                    "s3:GetBucketLocation",
+                    "s3:ListBucketMultipartUploads",
+                    "s3:ListBucketVersions"
+                ],
+                "Resource": "arn:aws:s3:::[BUCKET NAME]"
+            },
+            {
+                "Sid": "AllowUserManageBucketObjects",
+                "Effect": "Allow",
+                "Principal": {
+                    "AWS": "arn:aws:iam::[USER ARN]"
+                },
+                "Action": "s3:*",
+                "Resource": "arn:aws:s3:::[BUCKET NAME]/*"
+            }
+        ]
+    }
+
+
+After the bucket is set up, the bucket can be configured in a Wagtail project.
+
+Set up django-storages
+~~~~~~~~~~~~~~~~~~~~~~
+
+You need to install ``django-storages`` and ``boto3``.
+
+.. code:: sh
+
+   pip install django-storages[boto3]
+
+Set up your S3 bucket with ``django-storages``. The following code will allow
+configuration with environment variables.
+
+.. code:: python
+
+    # settings.py
+    import os
+
+
+    if "AWS_STORAGE_BUCKET_NAME" in os.environ:
+        # Add django-storages to the installed apps
+        INSTALLED_APPS = INSTALLED_APPS + ["storages"]
+
+        # https://docs.djangoproject.com/en/stable/ref/settings/#default-file-storage
+        DEFAULT_FILE_STORAGE = "storages.backends.s3boto3.S3Boto3Storage"
+
+        AWS_STORAGE_BUCKET_NAME = os.environ["AWS_STORAGE_BUCKET_NAME"]
+
+        # Disables signing of the S3 objects' URLs. When set to True it
+        # will append authorization querystring to each URL.
+        AWS_QUERYSTRING_AUTH = False
+
+        # Do not allow overriding files on S3 as per Wagtail docs recommendation:
+        # https://docs.wagtail.io/en/stable/advanced_topics/deploying.html#cloud-storage
+        # Not having this setting may have consequences in losing files.
+        AWS_S3_FILE_OVERWRITE = False
+
+        # Default ACL for new files should be "private" - not accessible to the
+        # public. Images should be made available to public via the bucket policy,
+        # where the documents should use wagtail-storages.
```

Comment:
Suggest this comment could be simplified to:

```
# Default ACL for new files should be "private".
# Images will then be explicitly be made public via the bucket policy,
# whereas the documents will use wagtail-storages.
```

Would need to check I haven't missed the point though!